### ISSUE TITLE
Amidst keeps crashing.

### ISSUE THREAD

**Issue by glokuosta** [toolbox4minecraft/amidst#151]
I was never able to run amidst for once in my life. Neither on this computer or my old one. I used old versions, newest version, another thing named amidst exporter, but they all gave me errors. 

```
[info] Amidst v4.1
[info] Current system time: 2016-04-12 23:19:22.405
[info] System.getProperty("os.name") == 'Windows 10'
[info] System.getProperty("os.version") == '10.0'
[info] System.getProperty("os.arch") == 'amd64'
[info] System.getProperty("java.version") == '1.8.0_77'
[info] System.getProperty("java.vendor") == 'Oracle Corporation'
[info] System.getProperty("sun.arch.data.model") == '64'
[info] Not using OpenGL.
[info] Biome profile activated: default
[info] using '.minecraft' directory at: 'C:\Users\X\AppData\Roaming\.minecraft', libraries: 'C:\Users\X\AppData\Roaming\.minecraft\libraries'
[info] using biome profiles at: 'biome'
[info] Scanning for profiles.
[info] Successfully loaded profile list.
[info] Beginning latest version list load.
[info] Attempting to download remote version list...
[info] Successfully loaded version list. URL: https://launchermeta.mojang.com/mc/game/version_manifest.json
[warning] cannot find valid version directory for launcher profile 'Forge'
[warning] cannot find valid version directory for launcher profile '1.8-ShadersMod'
[warning] cannot find valid version directory for launcher profile '1.7 forge'
[info] using minecraft launcher profile '1.9' with versionId '1.9.2'
[info] Loading libraries.
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\oshi-project\oshi-core\1.1\oshi-core-1.1.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\dev\jna\jna\3.4.0\jna-3.4.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\dev\jna\platform\3.4.0\platform-3.4.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\ibm\icu\icu4j-core-mojang\51.2\icu4j-core-mojang-51.2.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\sf\jopt-simple\jopt-simple\4.6\jopt-simple-4.6.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\codecjorbis\20101023\codecjorbis-20101023.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\codecwav\20101023\codecwav-20101023.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\libraryjavasound\20101123\libraryjavasound-20101123.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\librarylwjglopenal\20100824\librarylwjglopenal-20100824.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\soundsystem\20120107\soundsystem-20120107.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\io\netty\netty-all\4.0.23.Final\netty-all-4.0.23.Final.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\google\guava\guava\17.0\guava-17.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\commons\commons-lang3\3.3.2\commons-lang3-3.3.2.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\commons-io\commons-io\2.4\commons-io-2.4.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\commons-codec\commons-codec\1.9\commons-codec-1.9.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jinput\jinput\2.0.5\jinput-2.0.5.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jutils\jutils\1.0.0\jutils-1.0.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\google\code\gson\gson\2.2.4\gson-2.2.4.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\mojang\authlib\1.5.22\authlib-1.5.22.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\mojang\realms\1.8.15\realms-1.8.15.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\commons\commons-compress\1.8.1\commons-compress-1.8.1.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\commons-logging\commons-logging\1.1.3\commons-logging-1.1.3.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\logging\log4j\log4j-api\2.0-beta9\log4j-api-2.0-beta9.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\logging\log4j\log4j-core\2.0-beta9\log4j-core-2.0-beta9.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl\2.9.4-nightly-20150209\lwjgl-2.9.4-nightly-20150209.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl_util\2.9.4-nightly-20150209\lwjgl_util-2.9.4-nightly-20150209.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl-platform\2.9.4-nightly-20150209\lwjgl-platform-2.9.4-nightly-20150209-natives-windows.jar
[info] Skipping library: org.lwjgl.lwjgl:lwjgl:2.9.2-nightly-20140822
[info] Skipping library: org.lwjgl.lwjgl:lwjgl_util:2.9.2-nightly-20140822
[info] Skipping library: org.lwjgl.lwjgl:lwjgl-platform:2.9.2-nightly-20140822
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jinput\jinput-platform\2.0.5\jinput-platform-2.0.5-natives-windows.jar
[info] Recognised Minecraft Version 1.9 with the magic string "qwoomajzpb[Llp;mm[J[[Jmi".
[info] Reading 1.9.2.jar
[info] Jar load complete.
[info] Searching for classes...
[info] Found class GenLayer: axu
[info] Found class IntCache: axs
[info] Found class BlockInit: kn
[info] Found class WorldType: ahy
[info] Class search complete.
[info] Loading classes...
[info] unable to find the real class method axu.a(long, @WorldType) -> GenLayer.initializeAllBiomeGenerators2
[info] Classes loaded.
[info] Minecraft load complete.
[info] Not writing to seed history file, because it does not exist: history.txt
[info] Creating world with seed '2028644939958853256' and type 'Default'
[info] Using the following generator options: 
[crash] Amidst has encounted an uncaught exception on the thread Thread[AWT-EventQueue-0,6,main]
[crash] java.lang.ExceptionInInitializerError
	at amidst.mojangapi.world.WorldBuilder.create(WorldBuilder.java:113)
	at amidst.mojangapi.world.WorldBuilder.fromSeed(WorldBuilder.java:56)
	at amidst.mojangapi.MojangApi.createWorldFromSeed(MojangApi.java:122)
	at amidst.gui.main.Actions.newFromSeed(Actions.java:75)
	at amidst.gui.main.Actions.newFromRandom(Actions.java:67)
	at amidst.gui.main.menu.Menus.lambda$item$55(Menus.java:126)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.AbstractButton.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(Unknown Source)
	at amidst.ResourceLoader.getImage(ResourceLoader.java:26)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<init>(DefaultWorldIconTypes.java:36)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<clinit>(DefaultWorldIconTypes.java:20)
	... 44 more

```

**Comment by stefandollase** [comment]
Thanks for the report. It sounds like you ran into a really unlucky streak of bugs. Lets see how we can fix this.

I shortly looked into it, but was unable to reproduce the error. Please provide more information:

* Did you build Amidst by yourself, or did you download it directly from [https://github.com/toolbox4minecraft/amidst/releases](https://github.com/toolbox4minecraft/amidst/releases)?
* Do you use the `.jar` or `.exe` file?
* How do you execute Amidst?
* Did you apply any manual changes to the selected Minecraft version?
* Is there anything special, we should know about your setup? e.g. about the Java installation, Minecraft installation, Operating System ...

Also, please try the following:

* copy the Amidst jar file, the name of the copy should be `amidst.zip`, make sure there is no `.jar` at the end
* open the copied file with any zip program
* navigate to the directory `/amidst/gui/main/icon/`
* count the number of files in this directory, there should be 15 files in total, all have the `.png` file extension
* try to open all png files

**Comment by glokuosta** [comment]
Thanks for replying. 
I downloaded it from https://github.com/toolbox4minecraft/amidst/releases, Both java and exe versions give the same error. I just double click to run it. Tried random seed and it gave errors. I have also tried entering a seed but that's not working too. 

I have a couple of modded profiles in red color but I use unmodded profiles in the amidst so that shouldn't be a problem.

On my old pc, I was running on Windows XP 32 bit with 32 bit Java and now I am trying it on 64 bit Windows 10 with 64 bit Java 1.8.0 u77 installed.

I had checked all of the png files in the exe and the jar file but they all are working and there are 15 of them. I tried checking jar file for errors with 7zip and that returned no errors too.

I am using TeamExtreme Cracked launcher so I can get more than 1 account on cracked servers. I tried using premium mode and a premium account too and that still gives me the same error. 

And I just tried downloading original launcher and that gives the same error.

**Comment by Treer** [comment]
The log suggests you have some modded Minecraft profiles (forge, shaders etc).

I can't tell whether you've been trying to run Amidst on a modded profile, but that nearly always causes Amidst to fail, so it seemed worth mentioning - make sure you've tried Amidst on a new unmodified vanilla profile.

**Comment by stefandollase** [comment]
@glokuosta
Thanks for the additional information and testing. Sadly, this does not help to resolve the issue. Also, I currently don't have another idea how to resolve this.

I just added #156 to further investigate this issue. It will be included in the next release. It won't fix anything, but it will produce a more descriptive error message, that hopefully helps to resolve this issue.

> I was never able to run amidst for once in my life. Neither on this computer or my old one. I used old versions, newest version, another thing named amidst exporter, but they all gave me errors.

I would like to know whether it was always the same error message. Can you please post some of the other error messages?

**Comment by glokuosta** [comment]
@Treer I don't and can't choose modded profiles, Tried 1.8.9 and 1.9.2, and also tried downloading and running with original launcher, still gives the same error.
Actually, I first downloaded provided link at minecraft.net, that doesn't require me to download java and comes with its runtime, it gave some errors, then tried the second download link that requires java installed, that worked fine but amidst still crashed.
I have downloaded java directly from java.com, with offline installer for 64- bit, I don't have any other java installations.

@stefandollase I currently don't have access to my old computer, but if I can recall correctly, it was the same error. 
I had a Kali Linux installation on my hard drive, tried to run jar file from there, didn't expect it to run and it didn't. 
When I choose new from seed and new from random seed, error is still same, but when I choose open save game and choose my level.dat file, error changes a little bit. I'm going to post that too.
```
[crash] Amidst has encounted an uncaught exception on the thread Thread[AWT-EventQueue-0,6,main]
[crash] java.lang.ExceptionInInitializerError
	at amidst.mojangapi.world.WorldBuilder.create(WorldBuilder.java:113)
	at amidst.mojangapi.world.WorldBuilder.fromSaveGame(WorldBuilder.java:79)
	at amidst.mojangapi.MojangApi.createWorldFromSaveGame(MojangApi.java:140)
	at amidst.gui.main.Actions.openSaveGame(Actions.java:89)
	at amidst.gui.main.menu.Menus.lambda$item$55(Menus.java:126)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.AbstractButton.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: input == null!
	at javax.imageio.ImageIO.read(Unknown Source)
	at amidst.ResourceLoader.getImage(ResourceLoader.java:26)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<init>(DefaultWorldIconTypes.java:36)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<clinit>(DefaultWorldIconTypes.java:20)
	... 43 more
```

**Comment by stefandollase** [comment]
Thanks again. However, this does not help, either :-/ I looked at the diff of both error messages and the difference is really just that you clicked on *from save game* instead of *from random*.

I think we have to wait for the additional information that will be provided by the new release (see #156), but I really don't think this will provide any new information.

The new release is not yet scheduled, so this might take a while.

Do you have similar issues with other (Java) programs? I am still not sure that this is an issue with Amidst, since you are the only one that reports the issue. Amidst works fine for many other users, and the same is true for the old versions of Amidst.

**Comment by stefandollase** [comment]
I just created a new pre-release, which contains #156. Can you post the error message that is created by this release?

**Comment by glokuosta** [comment]
Nice idea about talking old versions :)
I have downloaded old versions from releases page, both jar and exe versions of them, and it seems this problem started at beta 3. Its description says "Most of the changes are ported from AmidstExporter." and I have almost the same problem with AmidstExporter. It still says "Thread[AWT-EventQueue-0,6,main]" but last line is 50 more and some other minor changes.

The problem with beta 2 and beta 1 is, it generates the world, with changing FPS values, coordinates, but It doesn't show anything. It is just pure black like this:
![amidst](https://cloud.githubusercontent.com/assets/18428755/14638131/7d84cd36-063e-11e6-8685-1078b67a60e4.PNG)

Also downloaded latest 32-bit Java and tried to run jar file from command line with 32 and 64 bit versions, nothing have changed.

**Comment by glokuosta** [comment]
Oh great timing.
Here it is:
```
[info] Amidst v4.2-beta1
[info] Current system time: 2016-04-19 15:13:44.659
[info] System.getProperty("os.name") == 'Windows 10'
[info] System.getProperty("os.version") == '10.0'
[info] System.getProperty("os.arch") == 'amd64'
[info] System.getProperty("java.version") == '1.8.0_77'
[info] System.getProperty("java.vendor") == 'Oracle Corporation'
[info] System.getProperty("sun.arch.data.model") == '64'
[info] Not using OpenGL.
[info] Biome profile activated: default
[info] using '.minecraft' directory at: 'C:\Users\X\AppData\Roaming\.minecraft', libraries: 'C:\Users\X\AppData\Roaming\.minecraft\libraries'
[info] using biome profiles at: 'biome'
[info] Scanning for profiles.
[info] Successfully loaded profile list.
[info] Beginning latest version list load.
[info] Attempting to download remote version list...
[info] Successfully loaded version list. URL: https://launchermeta.mojang.com/mc/game/version_manifest.json
[warning] cannot find valid version directory for launcher profile 'Forge'
[warning] cannot find valid version directory for launcher profile '1.7 forge'
[warning] cannot find valid version directory for launcher profile '1.8-ShadersMod'
[info] using minecraft launcher profile '1.9' with versionId '1.6.4'
[info] Loading libraries.
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\sf\jopt-simple\jopt-simple\4.5\jopt-simple-4.5.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\codecjorbis\20101023\codecjorbis-20101023.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\codecwav\20101023\codecwav-20101023.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\libraryjavasound\20101123\libraryjavasound-20101123.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\librarylwjglopenal\20100824\librarylwjglopenal-20100824.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\paulscode\soundsystem\20120107\soundsystem-20120107.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\argo\argo\2.25_fixed\argo-2.25_fixed.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\bouncycastle\bcprov-jdk15on\1.47\bcprov-jdk15on-1.47.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\google\guava\guava\14.0\guava-14.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\apache\commons\commons-lang3\3.1\commons-lang3-3.1.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\commons-io\commons-io\2.4\commons-io-2.4.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jinput\jinput\2.0.5\jinput-2.0.5.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jutils\jutils\1.0.0\jutils-1.0.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\com\google\code\gson\gson\2.2.2\gson-2.2.2.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl\2.9.0\lwjgl-2.9.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl_util\2.9.0\lwjgl_util-2.9.0.jar
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\org\lwjgl\lwjgl\lwjgl-platform\2.9.0\lwjgl-platform-2.9.0-natives-windows.jar
[info] Skipping library: org.lwjgl.lwjgl:lwjgl:2.9.1-nightly-20130708-debug3
[info] Skipping library: org.lwjgl.lwjgl:lwjgl_util:2.9.1-nightly-20130708-debug3
[info] Skipping library: org.lwjgl.lwjgl:lwjgl-platform:2.9.1-nightly-20130708-debug3
[info] Found library: C:\Users\X\AppData\Roaming\.minecraft\libraries\net\java\jinput\jinput-platform\2.0.5\jinput-platform-2.0.5-natives-windows.jar
[info] Recognised Minecraft Version 1.6.4 with the magic string "mvlv[Ljs;hn[J[J[J[J[J[[J".
[info] Reading 1.6.4.jar
[info] Jar load complete.
[info] Searching for classes...
[info] Found class GenLayer: akq
[info] Missing class BlockInit
[info] Found class IntCache: ako
[info] Found class WorldType: acg
[info] Class search complete.
[info] Loading classes...
[info] unable to find the real class method akq.a(long, @WorldType, String) -> GenLayer.initializeAllBiomeGenerators3
[info] Classes loaded.
[info] Minecraft load complete.
[info] Not writing to seed history file, because it does not exist: history.txt
[info] Creating world with seed '-3657921299583695162' and type 'Default'
[info] Using the following generator options: 
[crash] Amidst has encounted an uncaught exception on the thread Thread[AWT-EventQueue-0,6,main]
[crash] java.lang.ExceptionInInitializerError
	at amidst.mojangapi.world.WorldBuilder.create(WorldBuilder.java:114)
	at amidst.mojangapi.world.WorldBuilder.fromSeed(WorldBuilder.java:52)
	at amidst.mojangapi.MojangApi.createWorldFromSeed(MojangApi.java:118)
	at amidst.gui.main.Actions.newFromSeed(Actions.java:78)
	at amidst.gui.main.Actions.newFromRandom(Actions.java:70)
	at amidst.gui.main.menu.Menus.lambda$item$55(Menus.java:131)
	at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
	at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
	at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
	at javax.swing.AbstractButton.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
	at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$500(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: unable to load resource: '/amidst/gui/main/icon/vıllage.png'
	at amidst.ResourceLoader.getResourceURL(ResourceLoader.java:27)
	at amidst.ResourceLoader.getImage(ResourceLoader.java:34)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<init>(DefaultWorldIconTypes.java:35)
	at amidst.mojangapi.world.icon.type.DefaultWorldIconTypes.<clinit>(DefaultWorldIconTypes.java:20)
	... 44 more
```

**Comment by stefandollase** [comment]
Wow, I really did not expect this error message to resolve the issue, but it did: Your village is missing the i-dot ...

The cause of this is, that Java's `String.toLowerCase()` method uses the default `Locale`, so [this](https://github.com/toolbox4minecraft/amidst/blob/master/src/main/java/amidst/mojangapi/world/icon/type/DefaultWorldIconTypes.java#L39) causes the enum entry `VILLAGE` to produce the filename `/amidst/gui/main/icon/vıllage.png` on machines with a turkish `Locale`.

I will create a second pre-release to fix this.

**Comment by glokuosta** [comment]
So, the only relationship between my old and new computer is locale :P 
Waiting for the new release.

**Comment by stefandollase** [comment]
This is fixed by #158, which is included in the new pre-release.

I shortly looked into the origin of this issue and it was already included in the old Amidst by @skiphs. Thus, AmidstExporter also includes it. I am not sure why you did not experience the issue with the beta1 and beta2 of Amidst v4.0. It might be related to the other bugs that are included in these releases, e.g. the black/white screen issue.

**Comment by glokuosta** [comment]
It works now, thanks :+1:
